test/00-init.ts

```diff
@@ -44,4 +47,9 @@ Promise.resolve(fs.promises.readdir(__dirname))
 			require(`./${fileName}`);
 		});
 	})
+	.then(() =>
+		supertest(app)
```

Comment:
What's this for?

Reply:
This is needed to load the device types table in the database. Without it, on a single test file run, the application creation will fail with a 404 since the application types cannot be found.